test/distributed/org/apache/cassandra/distributed/test/accord/AccordCQLTest.java

```diff
@@ -2422,13 +2414,13 @@ private void testSelectUDTReference(String tableDDL) throws Exception
     @Test
     public void testSelectMultiCellUDTFieldReference() throws Exception
     {
-        testSelectUDTFieldReference("CREATE TABLE " + qualifiedTableName + " (k int PRIMARY KEY, customer person)");
+        testSelectUDTFieldReference("CREATE TABLE " + qualifiedTableName + " (k int PRIMARY KEY, customer person) WITH transactional_mode='" + transactionalMode + "'");
```

Comment:
This test is a strong argument in favor of allowing the default to be set even if it is a keyspace level thing :-)